lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -723,7 +723,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   ///    painting.
   @override
   void render(ui.Scene scene, [ui.FlutterView? view]) {
-    renderer.renderScene(scene);
+    renderer.renderScene(scene, view);
```

Comment:
If you decide to take my suggestion and make `view` a required param, then we can change this to:
```suggestion
    renderer.renderScene(scene, view ?? implicitView);
```